src/components/atoms/Button/Button.tsx

```diff
@@ -0,0 +1,10 @@
+import { StyledButton } from './Button.styles';
+import * as React from 'react';
+
+interface Props {
+	children: React.ReactNode;
+}
+
+export const Button = ({ children }: Props) => {
```

Comment:
import {ReactNode} from "react"

{ children } : { children: ReactNode }

1. Jeśli można zdestrukturyzować Reacta, to to róbmy.
2. Jeśli jest tylko jeden props, to nie róbmy interfejsu.